examples/vue-todos/src-tauri/src/main.rs

```diff
@@ -3,7 +3,7 @@
   windows_subsystem = "windows"
 )]
 
-use tauri::{MenuItem, Menu, Submenu};
+use tauri::{Menu, MenuItem, Submenu};
```

Comment:
yeah this is MUCH better; actually have no memory of reversing these but guess I did